modules/working-with-accelerator-profiles.adoc

```diff
@@ -0,0 +1,55 @@
+:_module-type: CONCEPT
+
+[id='working-with-accelerator-profiles_{context}']
+= Working with accelerator profiles
+
+[role='_abstract']
+To configure accelerators for your data scientists to use in {productname-short}, you must create an associated accelerator profile. An accelerator profile is a custom resource definition (CRD) on OpenShift that has an AcceleratorProfile resource, and defines the specification of the accelerator.  
+
+For accelerators that are new to your deployment, you must manually configure an accelerator profile for the accelerator. If your deployment contained NVIDIA GPUs before upgrading {productname-short}, an accelerator profile generates automatically after you upgrade to the latest version of {productname-short}, and resides in the `Instances` section of the `AcceleratorProfile` custom resource definition (CRD).
+
+If you add NVIDIA GPUs to your deployment after upgrading {productname-short}, or you add them to your deployment after you install a new version of the {productname-short} Operator, you must manually create an accelerator profile for your NVIDIA GPUs.
```

Comment:
1/ I think we should not mention NVIDIA GPU specifically in this doc unless we are giving some particular example. We should rather use some generic term like `accelerator`, `hardware accelerator`, `hardware accelerator unit` or similar?

2/ These two upgrade information - it's just a one time thing for the upgrade between two particular releases. We should put this note into some Upgrade Notes - that AcceleratorProfile will be created automatically if your deployment has had the hardware accelerator unit pre-configured before the upgrade already.

In general, here we should only have some message like this:

```
For hardware accelerators that are new to your deployment, you must manually configure an accelerator profile for that particular hardware accelerator. If your deployment contained a hardware accelerator before the upgrade, it will be preserved after the upgrade.
```

WDYT?

Reply:
I have reworded the parapgraph based on your suggestion. I have used the term "accelerators".

Reply:
I will update the downstream RHODS upgrade guides with information such as:

For hardware accelerators that are new to your deployment, you must manually configure an accelerator profile for that particular hardware accelerator. If your deployment contained a hardware accelerator before the upgrade, it will be preserved after the upgrade.

Reply:
Thanks! :slightly_smiling_face: